README.adoc

```diff
@@ -127,6 +127,19 @@ NOTE: Debug output from `vagrant-service-manager` is prepended with
 the following string:
 `DEBUG command: [ service-manager: <command name / log message> ]`
 
+=== Configurations
+
+- HTTP Proxy Settings +
+User can able to start ADB in proxy environment by specifying following configurations
```

Comment:
> I think I'd just call "HTTP Proxy configuration" and place it after "Ip address detection"

I kept it so that we can add more Vagrantfile configurations specifics docs in this section only. 
Like the openshift options configurations which are now only documented under [Using ADB with host-based tools](https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/using.adoc#using-adb-with-host-based-tools-eclipse-and-clis) point 3.